float64 angle
float64 angle_rate
uint8[3] acc